infrastructure/transform/main.tf

```diff
@@ -72,6 +72,10 @@ resource "databricks_cluster" "fixed_single_node" {
   ]
 }
 
+resource "databricks_secret_scope" "secrets" {
```

Comment:
Does this define a _built-in_ secret scope then? And it's what we're going to use (because kv backed scope does not support SP auth?)

Reply:
yes this is a built-in secret scope. The KV backed scopes can only be created by a user context, by a user with certain AAD permissions - so would never work for builds

Reply:
See https://learn.microsoft.com/en-us/azure/databricks/security/secrets/secret-scopes#--create-an-azure-key-vault-backed-secret-scope-using-the-databricks-cli for ref